### ISSUE TITLE
Find In Files in specific folder without opening the folder in VSCode

### ISSUE THREAD

**Issue by LiPengfei19820619** [Microsoft/vscode#17164]
- VSCode Version:
  1.8.0-Insider
- OS Version:
  Windows 10 64bit

Steps to Reproduce:

1. open a file in VSCode
2. Find In Files.
    I want to search some text in specific folder without opening the folder. But I find I can only search in the opening files, can not search in other files in the folder. I have to open the folder before searching some text in the folder.
    I hope VSCode can support find in specific folder without opening the folder in VSCode first.

    I can do this in almost all other editors, such as UE, Notepad++, Sublime Text.

Thank you!

**Comment by LiPengfei19820619** [comment]
@sandy081 
Would you please add this feature to the iteration plan? 
Thank you very much.

**Comment by LiPengfei19820619** [comment]
This is a very useful and essential feature in my work, so I hope it can appear in the next release.
Thanks very much!

**Comment by sandy081** [comment]
@LiPengfei19820619 Sorry, Not sure if this can be made in next release. We will prioritize it as per importance.

**Comment by LiPengfei19820619** [comment]
@sandy081  Thanks for your response.

**Comment by roblourens** [comment]
I think I will actually be able to do this for May. I wanted to do it in April and ran out of time.

**Comment by LiPengfei19820619** [comment]
@roblourens Hi, Would you be able to do this for June?
Thanks a lot.

**Comment by roblourens** [comment]
Maybe, not sure

**Comment by roblourens** [comment]
Can now search outside of the current workspace in 1.14:

![image](https://user-images.githubusercontent.com/323878/27654328-c64cd3ba-5bf6-11e7-87f6-20186ffd79c8.png)

**Comment by surjithctly** [comment]
@roblourens Is it possible to include a browse icon for  "files in include" textbox.? Currently, If I need to search for external folders, I had to manually copy the path and paste. So if there an option to browse the file or folder path, That would be super awesome.

I switched from Sublime yesterday and seems like this feature is missing in VSCode.

Thanks a lot for your consideration.

~Surjith
